.metadocrc

```diff
@@ -0,0 +1,12 @@
+{
+	"files": [
+		"lib/core.js",
```

Comment:
This was necessary, because `client.js` access global objects like `impress` or `api`, otherwise we cannot generate doc. I've thought, that only doc for client should be in this PR, so have just ignored doc for core. Should I add it?

Reply:
In this case, you can split commit into docs for `core` and `client` or create new PR. Also, it will be better to use `impress.js` since unlike `lib/core.js` it exports `impress`.

Reply:
AFAIK doc for `impress` will not be generated in this case, because the way of export is not compatible for `metadoc`

Reply:
It should be compatible since `metadoc` requires an object with methods which `impress` is. There shouldn't be problems with documentation generation for impress. If you found a bug or encountered unexpected behaviour, please describe it here or open an issue.